diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8f9a963..9a2cda3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -44,4 +44,4 @@ add_subdirectory(xtcdata)
 add_subdirectory(psana)
 add_subdirectory(psdaq)
 add_subdirectory(pgp)
-add_subdirectory(drp)
+# add_subdirectory(drp)
diff --git a/psdaq/CMakeLists.txt b/psdaq/CMakeLists.txt
index ef17a9e..7f7e2f2 100644
--- a/psdaq/CMakeLists.txt
+++ b/psdaq/CMakeLists.txt
@@ -1,10 +1,10 @@
 add_subdirectory(service)
 add_subdirectory(mmhw)
-add_subdirectory(cphw)
-add_subdirectory(eb)
-add_subdirectory(epicstools)
-add_subdirectory(dti)
-add_subdirectory(xpm)
-add_subdirectory(hsd)
-add_subdirectory(hdf5)
-add_subdirectory(app)
+# add_subdirectory(cphw)
+# add_subdirectory(eb)
+# add_subdirectory(epicstools)
+# add_subdirectory(dti)
+# add_subdirectory(xpm)
+# add_subdirectory(hsd)
+# add_subdirectory(hdf5)
+# add_subdirectory(app)
